### ISSUE TITLE
Multi-touch is not working when adding a second pointer.

### ISSUE THREAD

**Issue by levieggert** [photonstorm/phaser#4341]
## Version

* Phaser Version: v3.15.1

* Operating system: iPhone 7 11.4.1

* Browser: iOS Safari

## Description

I am working on a game that supports 2 touch pointers.  When the game is created a second pointer is added via this.input.addPointer().  This object exists when I console.log this.input.pointer2.

In an update method I am logging the following output for pointer1 and pointer2. 
this.input.pointer1.isDown
this.input.pointer1.x + ", " + this.input.pointer1.y
this.input.pointer1.downX + ", " + this.input.pointer1.downY
this.input.pointer2.isDown
this.input.pointer2x + ", " + this.input.pointer2.y
this.input.pointer2.downX + ", " + this.input.pointer2.downY

When I press 2 fingers down on the device screen I get true for pointer1.isDown and the positions update as I move my finger.  pointer2.isDown evaluates to false and positions stay at 0, 0.  The object exists, but isDown remains false and I am unable to get the touch position.

Is something else required to enable mult-touch or is this a bug?

## Example Test Code

import Phaser from 'phaser'
import { Game } from '../game'

export class GameScene extends Phaser.Scene
{
    debugText = null

    constructor()
    {
        super({key: Game.gameScene,
            physics: {
                default: 'arcade',
                arcade: {
                    debug: true,
                    fps: 60,
                    gravity: {x: 0, y: 0}
                }
            }
        })

        console.log("GameScene: constructor()")
    }

    preload(): void
    {
        console.log("GameScene: preload()")
    }

    create(): void
    {
        console.log("GameScene: create()")

        if (this.input.pointer2 == null || this.input.pointer2 == undefined)
        {
            this.input.addPointer()
        }

        // - debug text
        this.debugText = this.add.text(16, 16, '', {fill: '#7fdbff', fontFamily: 'monospace', lineSpacing: 4})
    }

    update(time, delta): void
    {
        // - debug text
        let debugText = "debug text"
        debugText = debugText + "\n" + "  pointer1 is down: " + this.input.pointer1.isDown
        debugText = debugText + "\n" + "  pointer1 position: " + this.input.pointer1.x + ", " + this.input.pointer1.y
        debugText = debugText + "\n" + "  pointer1 down position: " + this.input.pointer1.downX + ", " + this.input.pointer1.downY

        if (this.input.pointer2 != null && !this.input.pointer2.undefined)
        {
            debugText = debugText + "\n" + "  pointer2 is down: " + this.input.pointer2.isDown
            debugText = debugText + "\n" + "  pointer2 position: " + this.input.pointer2.x + ", " + this.input.pointer2.y
            debugText = debugText + "\n" + "  pointer2 down position: " + this.input.pointer2.downX + ", " + this.input.pointer2.downY
        }

        debugText = debugText + "\n" + "  pointer1: " + this.input.pointer1
        debugText = debugText + "\n" + "  pointer2: " + this.input.pointer2

        this.debugText.setText(debugText)
    }
}

**Comment by photonstorm** [comment]
Your code looks fine to me. I just tested the following on 3.15.1 and 3.16.0 and it worked on both of them on an iPad in Safari (real device, not emulation) - what device did you test it on?

```
var config = {
    type: Phaser.AUTO,
    parent: 'phaser-example',
    width: 800,
    height: 600,
    scene: {
        create: create,
        update: update
    }
};

var graphics;
var text;

var game = new Phaser.Game(config);

function create ()
{
    graphics = this.add.graphics();

    //  We need 3 extra pointers, as we only get 1 by default
    this.input.addPointer(3);

    text = this.add.text(10, 10, 'Use up to 4 fingers at once', { font: '16px Courier', fill: '#00ff00' });
}

function update ()
{
    if (this.input.pointer1.isDown || this.input.pointer2.isDown || this.input.pointer3.isDown || this.input.pointer4.isDown)
    {
        graphics.clear();
    }

    text.setText([
        'pointer1.isDown: ' + this.input.pointer1.isDown,
        'pointer2.isDown: ' + this.input.pointer2.isDown,
        'pointer3.isDown: ' + this.input.pointer3.isDown,
        'pointer4.isDown: ' + this.input.pointer4.isDown
    ]);

    graphics.fillStyle(0xff0000, 1);
    graphics.fillRect(this.input.pointer1.x, this.input.pointer1.y, 64, 64);

    graphics.fillStyle(0x00ff00, 1);
    graphics.fillRect(this.input.pointer2.x, this.input.pointer2.y, 64, 64);

    graphics.fillStyle(0x0000ff, 1);
    graphics.fillRect(this.input.pointer3.x, this.input.pointer3.y, 64, 64);

    graphics.fillStyle(0xff00ff, 1);
    graphics.fillRect(this.input.pointer4.x, this.input.pointer4.y, 64, 64);
}
```

**Comment by levieggert** [comment]
Hey thank you for testing so quickly.  I'm running on an iPhone 7 version 11.4.1.  

FYI: I also opened the console to see if any errors were showing up and no errors.  Any ideas for debugging this issue?

**Comment by levieggert** [comment]
Console is showing v3.15.1.  I'm using yarn to manage phaser.

<img width="1114" alt="console" src="https://user-images.githubusercontent.com/3237018/52238863-12163d00-289b-11e9-9916-448b1e391472.png">

**Comment by levieggert** [comment]
Here is what I see on my device when two fingers are down.

![screenshot](https://user-images.githubusercontent.com/3237018/52239039-7cc77880-289b-11e9-9f67-8509caad79aa.png)

**Comment by levieggert** [comment]
What happens when you run against here? 
https://galaxyrace-staging.firebaseapp.com/

**Comment by photonstorm** [comment]
@levieggert I would test the code I posted above on your host and see what happens.

**Comment by levieggert** [comment]
Hey guys, thank you for the help.  The labs example and sample above are both working.

I see what is happening now.  In the sample code that I provided, on GameScreen: create() I check if this.input.pointer2 is null or undefined before adding a pointer.  On my Mac -> Chrome this.input.pointer2 evaluates to undefined.  When I checked on my iPhone, this.input.pointer2 is already an object, so the if statement to add a pointer is never triggered.  I wonder if that is a bug?  Should this.input.pointer2 be null or undefined since it hasn't been added?  What is the proper way of adding pointers? I feel like I should have a check there so a pointer isn't added when the GameScreen is created successive times.

On Mac -> Chrome this.input.pointer2 evaluates to undefined.
On iPhone 7 -> Safari this.input.pointer2 already exists so this.input.addPointer() is never called.
`if (this.input.pointer2 == null || this.input.pointer2 == undefined)
    {
        this.input.addPointer()
}
`
